tgui/packages/tgui/interfaces/Mecha/ModulesPane.tsx

```diff
@@ -477,16 +490,46 @@ const SnowflakeSyringe = (props) => {
         />
       </LabeledList.Item>
       <LabeledList.Item label={'Reagent control'}>
+        {analyzed_reagents.map((reagent) => (
+          <LabeledList.Item key={`${reagent}`} label={`${reagent}`}>
+            <Button
+              content={'Toggle Synthesizing'}
+              onClick={() =>
+                act('equip_act', {
+                  ref: ref,
+                  gear_action: `toggle_reagent_${reagent}`,
+                })
+              }
+            />
+          </LabeledList.Item>
+        ))}
+      </LabeledList.Item>
+      <LabeledList.Item>
         <Button
-          content={'View'}
+          content={'Purge All'}
```

Comment:
same, no content please. also with props like `content` here, you can just type strings without curly brackets. the exception to that rule is string interpolation. i know the rest of the file has this everywhere but it's incorrect